manifests/server/initdb.pp

```diff
@@ -105,12 +164,25 @@
           require => Exec['postgresql_initdb'],
         }
       }
+    } else {
+      # This runs the initdb command, we use the existance of the PG_VERSION
+      # file to ensure we don't keep running this command.
+      exec { 'postgresql_initdb':
+        command   => $initdb_command,
+        creates   => "${datadir}/PG_VERSION",
+        user      => $user,
+        group     => $group,
+        logoutput => on_failure,
+        require   => File[$require_before_initdb],
+      }
     }
   } elsif $encoding != undef {
     # [workaround]
-    # by default pg_createcluster encoding derived from locale
+    # by default pg_createcluster encoding/locale derived from locale
     # but it do does not work by installing postgresql via puppet because puppet
     # always override LANG to 'C'
+    # See:  https://projects.puppetlabs.com/issues/4695
```

Comment:
perhaps better to link to https://tickets.puppetlabs.com/browse/PUP-1191 instead